### ISSUE TITLE
E501 line too long is not reported on line ending with a backslash

### ISSUE THREAD

**Issue by akitada** [PyCQA/pycodestyle#319]
Apparently `pep8` is failing to detect E501 on a line ending with a backslash.

``` python
open('long_long_long_long_long_long_long_long_long_long', mode='w+', buffering=1024) \
    or ''
```

Tested with `1.6.0a0`, installed from master.

**Comment by markpeek** [comment]
I believe this commit (497f1db5d142ddb891504d2513d52c5f195c05ac) introduced a regression when using # noqa. Using this commit with a testsuite file containing:
```
#: Okay
a = '1234567890123456789012345678901234567890123456789012345678901234567890' or \
    '1234567890123456789012345678901234567890123456789012345678901234567890'  # noqa
```
results in:
```
test_checkers_testsuite (testsuite.test_all.Pep8TestCase) ... pep8/testsuite/E99.py:2:80: E501:1:80 line too long (81 > 79 characters)
FAIL
```
Using the previous commit (e73ce7d2001f9cd3d1d6acdd1040b7e6abca77fe) passes.

**Comment by IanLee1521** [comment]
Thanks for the report @markpeek , I pushed the test you suggested in commit 8b14a59. I'll see what I can do to find out what's going on.

**Comment by asottile** [comment]
this appears to no longer be an issue:

```console
$ pycodestyle --version
2.5.0
$ pycodestyle t.py
t.py:1:80: E501 line too long (86 > 79 characters)
```
